crates/polars-stream/src/morsel.rs

```diff
@@ -23,6 +23,15 @@ impl MorselSeq {
     pub fn new(seq: u64) -> Self {
         Self(seq.checked_mul(2).unwrap())
```

Comment:
Can we document why we multiply by 2 here?

Reply:
It is documented, in the comment right above it.

```rust
// TODO: use least significant bit to indicate 'last morsel with this
// sequence number'.
```

Reply:
Ah.. Ok, now I see it. ;)